### PR TITLE
Connetor for driveplayer.com #1082

### PR DESCRIPTION
**Pull request by saintego**
Connetor for driveplayer.com

### CODE REVIEW

File: connectors/v2/driveplayer.com.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+/* global Connector */
+Connector.playerSelector = '.jp-play';
```
Comment (alexesprit) [inline connectors/v2/driveplayer.com.js]: `Connector.playerSelector` should be defined as `.jp-audio`. Now it's just a play button selector.

File: connectors/v2/driveplayer.com.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	var text = $('.playing > .artist').text().trim();
+	return text || null;
+};
+
+Connector.getTrack = function () {
+	var text = $('.playing > .title').text().trim();
+	return text || null;
+};
+
+Connector.isPlaying = function () {
```
Comment (alexesprit) [inline connectors/v2/driveplayer.com.js]: This `isPlaying` function can be replaced to `Connector.playButtonSelector = '.jp-play';` definition. But the function is correct too, though.

Comment (saintego) [inline connectors/v2/driveplayer.com.js]: Thanks. I was inspired by bandzone.cz.js so I made it in their way

**Comment by alexesprit** [comment]
Looks nice now! :+1:

**Pull request merged**